lib/screens/stake_screen/stake_screen.dart

```diff
@@ -0,0 +1,279 @@
+import 'dart:ui';
```

Comment:
Das doppelte Kommentieren spar ich mir jetzt mal.
Anmerkungen bitte erst in https://github.com/deusfinance/mobile-app/pull/2 korrigieren.
Am besten arbeitest du erst dort alles ab. Sobald ich es da dann genehmige, merge ich das nochmal hier rein.